distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1600,7 +1600,7 @@ type Column
                 run_vectorized_binary_op self op_name as_vector expected_result_type=Value_Type.Boolean skip_nulls=False new_name=result_name
             False ->
                 set = Set.from_vector as_vector error_on_duplicates=False
-                run_unary_op self set.contains new_name=result_name skip_nulls=False expected_result_type=Value_Type.Boolean
+                run_unary_op self set.contains_relational new_name=result_name skip_nulls=True expected_result_type=Value_Type.Boolean
```

Comment:
@radeusgd I'm not sure how to invoke the fallback implementation of `==` (line 214) -- what case uses this?

Reply:
I need more context for this question. Line 214 of which file?

Reply:
Try using `==` on a `Mixed` column, e.g.
```
t = Table.new [["X", ["a", 42, Nothing]], ["Y", ["b", 42, True]]]
(t.at "X") == (t.at "Y")
```

I think it should trigger it, but you can add some println to make sure :)

Reply:
Ah, I remember now -- `==` uses skip_nulls=True, so the fallback is never used for nulls.

Reply:
Oh :D